src/index.js

```diff
@@ -1,43 +1,192 @@
+});
+
+document.addEventListener('keydown', (press) => {
+  if (press.key === 'Enter' && addTaskInput.value) {
+    addTask.click();
+  }
+});
+
+const SteerChecked = () => {
+  const StoreCheck = JSON.parse(localStorage.getItem('tasks'));
+  const checkBoxs = document.querySelectorAll('.check');
+  StoreCheck.forEach((a, i) => {
+    if (a.completed === true) {
+      checkBoxs[i].checked = true;
+      document.getElementById(`ptask${i}`).style.textDecoration = 'line-through rgb(68, 68, 68)';
+    }
+  });
+};
+
+const removeChecked = () => {
+  const buttonRemove = document.getElementById('button-clear');
+  const checkBoxs = document.querySelectorAll('.check');
+  const TasksR = JSON.parse(localStorage.getItem('tasks'));
+  checkBoxs.forEach((check, i) => {
+    check.addEventListener('click', () => {
+      if (TasksR[i].completed === true) {
+        TasksR[i].completed = false;
+        checkBoxs[i].checked = false;
+        document.getElementById(`pTask${i}`).style.textDecoration = 'none';
+        localStorage.setItem('tasks', JSON.stringify(TasksR));
+      } else {
+        TasksR[i].completed = true;
+        checkBoxs[i].checked = true;
+        document.getElementById(`ptask${i}`).style.textDecoration = 'line-through rgb(68, 68, 68)';
+        localStorage.setItem('tasks', JSON.stringify(TasksR));
+      }
+    });
+
+    buttonRemove.addEventListener('click', () => {
+      const TaskFiltered = TasksR.filter((task) => task.completed !== true);
+      localStorage.setItem('tasks', JSON.stringify(TaskFiltered));
+      window.location.reload();
+    });
+  });
 };
 
-listDisplay();
\ No newline at end of file
+function updateId() {
+  const TasksR = JSON.parse(localStorage.getItem('tasks'));
+  TasksR.forEach((a, i) => {
+    a.index = i;
+    localStorage.setItem('tasks', JSON.stringify(TasksR));
+  });
+}
+
+function removeTask() {
+  const removeList = document.querySelectorAll('.hide1');
+  const Taskstored = JSON.parse(localStorage.getItem('tasks'));
+  removeList.forEach((a, i) => {
+    document.getElementById(`idBtn${i}`).addEventListener('click', () => {
+      const TaskFiltered = Taskstored.filter((task) => task.index !== i);
+      localStorage.setItem('tasks', JSON.stringify(TaskFiltered));
+      window.location.reload();
+      updateId();
+    });
+  });
+}
+
+function UpdateLabel() {
+  const ArrayStored = localStorage.getItem('tasks');
+  const ArrayStoredParse = JSON.parse(ArrayStored);
+  const inputs = document.querySelectorAll('.UpdateLabel');
+
+  inputs.forEach((element) => {
+    element.addEventListener('keypress', (event) => {
+      if (event.key === 'Enter') {
+        event.preventDefault();
+        ArrayStoredParse.forEach((a, i) => {
+          const listUpdate = document.getElementById(`updt${i}`);
+          ArrayStoredParse[i].description = listUpdate.value;
+          localStorage.setItem('tasks', JSON.stringify(ArrayStoredParse));
+          window.location.reload();
+        });
```

Comment:
- [ ] Trying to edit a task description, results in deleting all the other descriptions as shown below: 
![image](https://user-images.githubusercontent.com/61377322/166109531-597d89d3-bfa4-4be7-bc16-84bb43ca7130.png)

-**Hint**: As you can see in your code, for every edit task triggered,  line 176-180 is iterating through all the tasks description and sets the value to `listUpdate.value`(which is null as the other tasks are not being modified).